es-spatial-plugin/src/cmr/elasticsearch/plugins/spatial/script/helper.clj

```diff
@@ -1,13 +1,14 @@
-(ns cmr.es-spatial-plugin.spatial-script-helper
-  (:require [cmr.spatial.serialize :as srl]
-            [clojure.string :as s]
-            [cmr.common.util :as u])
-  (:import org.elasticsearch.index.fielddata.ScriptDocValues$Doubles
-           org.elasticsearch.search.lookup.DocLookup
-           org.elasticsearch.search.lookup.FieldsLookup
-           org.elasticsearch.search.lookup.FieldLookup
-           org.elasticsearch.common.logging.ESLogger))
-
+(ns cmr.elasticsearch.plugins.spatial.script.helper
+  (:require
+   [clojure.string :as s]
+   [cmr.common.util :as u]
+   [cmr.spatial.serialize :as srl])
+  (:import
```

Comment:
We have a different style for imports now. I think something like 
`(org.elasticsearch.search.lookup DocLookup FieldLookup FieldsLookup)`

Reply:
Done.